### PR TITLE
Add KalarChain

### PR DESCRIPTION
**Pull request by KalarChain**
Thanks for your support

### CODE REVIEW

File: _data/chains/eip155-1379.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://kalarchain.tech",
+  "shortName": "klc",
+  "chainId": 1379,
+  "networkId": 1379,
+  "icon": "kalarchain",
+  "explorers": [{
+    "name": "kalarscan",
+    "url": "https://explorer.kalarchain.tech/",
```
Comment (ligi) [inline _data/chains/eip155-1379.json:20]:
```suggestion
    "url": "https://explorer.kalarchain.tech",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Review by ligi — CHANGES_REQUESTED** [review]
I could not resolve your CID with `ipfs get` in a reasonable time - and before having a back and forth here I require you to put it in iconDownload. Before this back and forth did cost me a lot of time that I do not have.

**Comment by KalarChain** [comment]
Im so sorry to have wasted your precious time, I will fix it soon and be
more careful. Thank you very much.

Regards

Vào 01:09 AM, T.7, 29 Th7, 2023 ligi ***@***.***> đã viết:

**Comment by KalarChain** [comment]
> 

Hi Ligi, i checked the IPFS address in both on cloudflare and found it working fine. This error can also occur when the connection to cloudflare is unstable. You can do a quick check here:
For KalarChain icon

- https://cloudflare-ipfs.com/ipfs/bafkreihfoy2kgf2rebaoicso7z5h7ju46z6gtr64mskkths3qbfkrtnkjm

For KalarScan icon

- https://cloudflare-ipfs.com/ipfs/bafkreidbzzv3lrwlzquhlpl4jcmr54sifhi5ufoek6olb5kjh3h3kvktdq

**Review by ligi — APPROVED** [review]

**Pull request merged**